prow/cluster/starter/deck.deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: deck
+  labels:
+    app: deck
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
```

Comment:
This parameter allows for unavailability in deck. You can either use a more sane default since examples tend to be reused as is like 25%-25% (maxUnavailable is rounded down) or switch apiVersion to apps/v1beta1 and remove the strategy section altogether (it will default to rolling+25%-25% on creation).

Reply:
All of these are derived from `prod/cluster/starter.yaml`.  I dropped this (and `hook`) to a single replica when I was debugging problems scheduling everything on a too-small cluster.  I bumped them back to 2.